### ISSUE TITLE
Attend DX indicators sessions

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2991]

**Comment by cthomasONRR** [comment]
Accessibility 11/28/23
	• Requirements: 
		○ WCAG2.0 - level A and AA, 
		○ DHS's Trusted Tester, 
		○ M-23-22.
	• Focus on automated indicators
		○ Acknowledge that automated is not perfect, but low hanging fruit
	• WebAIM: 96% of all accessibility errors call in 6 categories:
		○ Low contrast text
		○ Missing alt text
		○ Empty links
		○ Mission form input labels
		○ Empty buttons
		○ Missing document language
	• Current indicators: 
		○ Color contrast, 
		○ HTML attribute, 
		○ Missing image description
	• Desired indicators - Priority/Severity Level - Level of Effort to fix
		Desired Indicators	Priority/Severity Level 	Level of Effort to fix
		Low contrast text/graphics	7/10	
		Missing alt text	5-8/10 important	
		Empty links 		
		Mission form input labels		
		Empty buttons		
		Missing document language 		
		(language not defined as "English")
		Font size & type	8/10	
		Captioning/Subtitles	8-10/10	
		Reading order	8/10	
		Tagged PDF		
		Digitization of forms & use of PDFs		Digital tools for this
		(scan for a particular button)
		Videos without audio descriptions		
		Keyboard navigation		More manual

**Comment by cthomasONRR** [comment]
@eelzi-ONRR @lpgoldstein I took notes on the first accessibility session, but not sure if the session in general was useful. Let me know if I should post them anywhere on Sharepoint.

**Comment by AlexandracmONRR** [comment]
I attended the Mobile/SEO session, below is the majority of the discussion:

Current way OCIO captures Mobile access:

- Viewport main tag - (BO field in csv) - "False" if not good mobile viewing
- OG Article Published Final URL (AP to AR and AT field in csv)
- Robots.txt is detected (comment: a lot of search engines see the top level but not the sub domains)
- Sitemap.xml is detected


What should they be scanning for Mobile/SEO experience?
Possible indicator categories:

- Page load is not fast enough on mobile networks
- Content is not sized correctly for the viewport
- Viewport not set
- Content wider than screen
- Does not have a meta viewport tag
- Does not provide fallback content when JS is not available
- Testing on devices (requirements to do this) - what mobiles are we expected to test on?
- Less reliance on PDF for regulations, use HTML to store information
- Text too small to read
- Clickable elements too close together
- OG Title, Description
- Inclusion of schema.org properties
- [flagged as content category to be covered in 2024] Strengthen language to not duplicate others' information (general content duplication instead of linking out).
